cc_utilities/command_line/sync_redcap_to_commcare.py

```diff
@@ -99,30 +100,20 @@ def main_with_args(
         if len(redcap_records.index) == 0:
             logger.info("No records returned from REDCap; aborting sync.")
         else:
-            cases_df, contacts_df = (
+            cases_df = (
                 redcap_records.pipe(collapse_checkbox_columns)
                 .pipe(normalize_phone_cols, phone_cols)
-                .pipe(split_cases_and_contacts, external_id_col)
+                .pipe(set_external_id_column, external_id_col)
             )
             logger.info(
                 f"Uploading {len(cases_df.index)} found patients (cases) to CommCare..."
             )
-            upload_data_to_commcare(
-                cases_df,
-                commcare_project_name,
-                "patient",
-                "external_id",
-                commcare_user_name,
-                commcare_api_key,
-                create_new_cases="off",
-                search_field="external_id",
+            upload_complete_records(
+                cases_df, commcare_api_key, commcare_project_name, commcare_user_name
```

Comment:
Optional nitpick: for me it would be a little easier to understand if the complete/incomplete split happened side by side in its own function, but I can see from the tests that this works as intended!